lib/game_stats.rb

```diff
@@ -1,4 +1,94 @@
 
 
+
+
 module GameStats
+
+	def highest_total_score(season) # integer
+		if away_winners(season) > home_winners(season)
+			away_winners(season)
+		elsif away_winners(season) < home_winners(season)
+			home_winners(season)
+		end
+	end
+
+	def lowest_total_score(season)
+		if away_winners(season) < home_winners(season)
+			away_winners(season)
+		elsif away_winners(season) > home_winners(season)
+			home_winners(season)
+		end
+	end
+
+	def get_games(season) # helper
```

Comment:
I might have missed something but I don't think #get_games was used anywhere.

Reply:
You're right, this helper method was made at the beginning but I ended up not using it so I will get rid of it once I'm sure all the other methods are working the way theyre intended to.